recipes/dials/meta.yaml

```diff
@@ -0,0 +1,87 @@
+{% set name = "dials" %}
+{% set version = "3.10.dev1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/dials/dials/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 572b1ce7366f8bde14a4e6df9649e426b6dd23a4ee16d2ab3e35151300dd9551
+  folder: dials
+
+build:
+  skip: True  # [win]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - cmake
+    - ninja
+  host:
+    - dxtbx
+    - msgpack-c
+    - pip
+    - boost
+    - boost-cpp
+    - cctbx-base
+    - python
+  run:
+    - {{ pin_compatible('boost') }}
+    - boost-cpp
+    - {{ pin_compatible('cctbx-base') }}
+    - cctbx
+    - {{ pin_compatible('dxtbx') }}
+    - gemmi
+    - h5py
+    - jinja2
+    - matplotlib-base
+    - {{ pin_compatible('numpy') }}
```

Comment:
Is this compiling against numpy or just using numpy?
If it is compiling against, then should add numpy to host dependencies for completeness sake.

Also alphabetical sorting of dependencies wouldn't go amiss 😉

Reply:
(And if only importing then just list it as `numpy`)

Reply:
That's a a good point, I don't think dials uses it at compile time, only dxtbx and cctbx. I think I left the pin_compatible _for_ these dependencies, but imagine that their restrictions would work.

Also, I'd tried to make it package-alphabetical but it's probably cleaner to do it in a way that can be simply sorted.